server/src/main/java/info/doseamigos/CommonModule.java

```diff
@@ -0,0 +1,14 @@
+package info.doseamigos;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.google.inject.AbstractModule;
+
+/**
+ * Created by jking31cs on 7/4/16.
```

Comment:
Need to add a better javadoc, this is used for providing an ObjectMapper to each handler.